docker/polkastats-backend/docker-compose.yml

```diff
@@ -59,6 +59,7 @@ services:
     command: /usr/app/polkastats-backend-v3/crawlers/block-listener.js    
     depends_on:
       - "postgres"
+      - "substrate-node"
```

Comment:
To prevent dependencies from actually being in service, we can use this check. Do not start the next service until the facility is up and running and in good health. Example :-)
```
     depends_on:
       postgres:
         condition: service_healthy
```

Reply:
I love this solution!  Let's use it.

Reply:
Umm... @melozo  that condition is also inside each service??  I am confused now.
Not sure how to use it. Could you push a commit?

Reply:
I think it will be in all the crawlers, we must wait to substrate this up and running, if not give errors in the execution. In this example we can apply:

```yaml
    depends_on:
      postgres:
        condition: service_healthy
      substrate-node:
        condition: service_healthy
```

Reply:
Finally they stopped supporting this flag. So this PR could be merge as it is.

Ref:
https://github.com/peter-evans/docker-compose-healthcheck/issues/3#issue-257120693
https://docs.docker.com/compose/compose-file/#depends_on